certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -62,6 +62,14 @@
     ['#', " } # managed by Certbot"],
 ]
 
+PREVIOUS_SSL_OPTIONS_HASHES = [
+    '0f81093a1465e3d4eaa8b0c14e77b2a2e93568b0fc1351c2b87893a95f0de87c',
```

Comment:
figured it out. we used to merge commits, not squash them, so going to where it was changed then `reset`ing to get the hash from right before then is inaccurate.